app.rb

```diff
@@ -1,9 +1,31 @@
 require 'sinatra/base'
+require 'sinatra/reloader'
+require 'pg'
+require './lib/peep'
+require './database_connection_setup'
 
 class Chitter < Sinatra::Base
+  configure :development do
+    register Sinatra::Reloader
+  end
+
   get '/test' do
     'Test page'
   end
 
+  get '/' do
+    erb :index
+  end
+
+  get '/peeps' do
```

Comment:
Really cleanly defined routes, with all of the business logic kept to the Peep model. Nice work on this.